ui/main.qml

```diff
@@ -220,6 +225,22 @@ StatusWindow {
         }
     }
 
+    // On MacOS, explicitely restore the window on activating
+    Connections {
+        target: Qt.application
```

Comment:
you could actually add a hint here:
```qml
enabled: Qt.platform.os === Constants.mac
```

Reply:
![image](https://github.com/status-im/status-desktop/assets/25482501/fb9943aa-57c4-4e85-98e8-8436613d6bc3)